OpenRA.Game/Player.cs

```diff
@@ -181,6 +182,24 @@ public bool HasFogVisibility
 			}
 		}
 
+		public Color PlayerStanceColor(Actor a, bool tan = false)
```

Comment:
I would suggest renaming `tan` to `isForSelectionBar` or something like that, to indicate that the intention is to tweak the colour for the bar.